views.py

```diff
@@ -0,0 +1,169 @@
+from flask import render_template, session, redirect, request
+from datetime import datetime
+
+from init import db, app
+from forms import LoginForm, RegistrationForm, OrderForm
+from tables import User, Dish, Category, Order
+from words import correct_word
+
+
+@app.route("/")
+def render_main():
+    items = db.session.query(Dish)
+    categories = db.session.query(Category)
+    session["word"] = correct_word(len(session.get("cart", [])))
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    return render_template("main.html", categories=categories, items=items, dishes=session.get("cart", []), user=user)
+
+
+@app.route("/cart/", methods=["GET", "POST"])
+def render_cart():
+    flag = 0
+    form = OrderForm()
+    cart = session.get("cart", [])
+    use_it = set()
+    new_cart = []
```

Comment:
why can't we just roll with `cart`?

Reply:
I guess, that we can, but I tried many different ways and the first one that worked - this one

Reply:
please rework this whole piece, it's just ugly and unneeded